src/functions-and-arrays.js

```diff
@@ -1,19 +1,35 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 < num2) return num2
+   else 
+   return num1
+  }
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(arr) {
+  let longestWord = arr[0]
+for (let words of arr) {
+  if (words.length > longestWord.length) {
+    longestWord = words
+  }
+}
+return longestWord;
+}
 
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+function sumNumbers(arrOfNum) {
+  let sum = arrOfNum[0];
+  for (let num of arrOfNum) {
+    sum += num
+  }
+  return sum;
+}
```

Comment:
I think here the function will add the first element twice. Since sum is first initialized with the first element, then the for loop starts with the first element.

Since the test cases expect you to return 0 if the array is empty, it is better to initialize the sum with 0, if the array is empty, the for-loop will not iterate and the function will just return 0.